docs/docs/guides/labs/components/building-pipelines-with-components/pyproject-toml.md

```diff
@@ -1,58 +0,0 @@
----
-title: 'pyproject.toml settings'
-sidebar_position: 500
----
-
-import Preview from '@site/docs/partials/\_Preview.md';
-
-<Preview />
-
-`pyproject.toml` contains `tool.dagster` and `tool.dg` sections:
-
-<CodeExample
-  path="docs_snippets/docs_snippets/guides/components/index/4-pyproject.toml"
```

Comment:
Do we need to remove the docs_snippets themselves too?

Reply:
I was thinking that but this guide doesn't even have its own snippets, these are links to another guide's snippets

Reply:
ah gotcha!